e2e/playwright/temporary-workspace.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { expect, test } from '@e2e/playwright/zoo-test'
+import { stringToBase64 } from '@src/lib/base64'
+
+test.describe('Temporary workspace', () => {
+  test(
+    'Opening a share link creates a temporary environment that is not saved.',
```

Comment:
Nitpick, but none of our test names end with periods.

```suggestion
    'Opening a share link creates a temporary environment that is not saved',
```